pygmt/tests/test_grdimage.py

```diff
@@ -205,10 +201,11 @@ def test_grdimage_central_meridians(grid, proj_type, lon0):
 # Cylindrical Equidistant (Q) projections plotted with xarray and NetCDF grids
 # are still slightly different with an RMS error of 25, see issue at
 # https://github.com/GenericMappingTools/pygmt/issues/390
-# TO-DO remove tol=1.5 and pytest.mark.xfail once bug is solved in upstream GMT
-@check_figures_equal(tol=1.5)
+# TO-DO remove pytest.mark.xfail once bug is solved in upstream GMT
+#       combine with test_grdimage_central_meridians_and_standard_parallels_fix
+@check_figures_equal()
 @pytest.mark.parametrize("lat0", [0, 30])
-@pytest.mark.parametrize("lon0", [0, 123, 180])
+@pytest.mark.parametrize("lon0", [123, 180])
 @pytest.mark.parametrize("proj_type", [pytest.param("Q", marks=pytest.mark.xfail), "S"])
```

Comment:
> A suggestion for a more clever way to do this with pytest parameterize would be welcome.

A lot more lines, but saves us from having to create another (very similar) unit test.

```suggestion
@pytest.mark.parametrize(
    ("proj_type", "lon0"),
    [
        ("Q", 0),
        pytest.param("Q", 123, marks=pytest.mark.xfail),
        pytest.param("Q", 180, marks=pytest.mark.xfail),
        ("S", 0),
        ("S", 123),
        ("S", 180),
    ],
)
```